app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ReferrerConfig.scala

```diff
@@ -20,7 +20,11 @@ import cats.data.NonEmptyList
 import play.api.libs.json.{ Format, Json }
 import uk.gov.hmrc.gform.sharedmodel.LocalisedString
 
-case class ReferrerConfig(allowedReferrerUrls: NonEmptyList[ReferrerUrlPattern], exitMessage: LocalisedString)
+case class ReferrerConfig(
+  allowedReferrerUrls: NonEmptyList[ReferrerUrlPattern],
+  title: Option[LocalisedString],
+  exitMessage: LocalisedString
```

Comment:
1. I don't see any logic related to translation of these two new fields.
2. Why not `SmartString`?